### DISCUSSION TITLE
Gmod wiki info outdated

### DISCUSSION THREAD

**Discussion by hoksalot** [WilliamVenner/vscode-glua-enhanced#38] [Q&A]
Is it possible to make the extension pull the latest information from the gmod wiki? Some function descriptions are outdated for me.

**Comment by WilliamVenner** [comment]
It already does

https://venner.io/gmod-wiki.json

**Reply by hoksalot** [reply]
Weird, the json file you linked is indeed up to date, however if the extension is using that file, how is it possible that some function descriptions are outdated for me anyway? (This is a fresh install of the extension, I don't think there's any sort of caching issue here)

**Reply by hoksalot** [reply]
If I understand this line correctly, the extension only pulls an update if the local wiki is out of date by over **1000 days**
https://github.com/WilliamVenner/vscode-glua-enhanced/blob/c5fd1ad013caf6e31749bfabf4f9064496fb7388/src/wikiProvider.js#L23 
I also tried manually updating `resources/wiki.json` from the link you provided, and yet it shows outdated info, as if it's not even using that file. This is so weird.
